apollo-router/src/configuration/mod.rs

```diff
@@ -189,7 +189,7 @@ impl Configuration {
             Value::Bool(true),
         );
         self.plugins.plugins.as_mut().unwrap().insert(
-            "experimental.include_subgraph_errors".to_string(),
+            "apollo.include_subgraph_errors".to_string(),
```

Comment:
Hmmm I think now it's an apollo one, it's no longer located in `self.plugins` but in `self.apollo_plugins` isn't it ?

Reply:
Ok. Re-reading. I think this was already in the wrong place. `experimental` plugins should be in `apollo_plugins`. Anyway, it definitely needs to be fixed.

Reply:
I think it is in the wrong place, but the processing of mandatory plugins positions it correctly. We should still fix this at some point.

Reply:
see: #1777